src/commands/skullboard.py

```diff
@@ -150,3 +167,97 @@ async def edit_or_send_skullboard_message(
+        self.db = SkullboardDB()
+        self.skullboard_channel_id = int(str(os.environ.get("SKULLBOARD_CHANNEL_ID")))
+
+    @app_commands.command(name="about", description="Learn about the Skullboard")
+    async def about(self, interaction: Interaction):
+        skullboard_info = (
+            "## 💀 WELCOME TO THE SKULLZONE 💀\n\n"
+            "The **Skullboard** is a fun way to track popular posts and active users in the CS Club! 💀\n"
+            f"When a post receives a certain number of 💀 reactions, it gets added to <#{self.skullboard_channel_id}>. 💀\n"
+            "Users earn a 💀 for their popular posts, and these 💀 contribute to their overall ranking. 💀\n"
+        )
+        cmds = [
+            "`/skull rank` to view current user rankings",
+            "`/skull hof` to view the Hall of Fame",
+        ]
+        cmds = "\n".join(cmds)
+
+        embed = Embed(colour=LIGHT_GREY)
+        embed.add_field(name="Commands:", value=cmds)
+        await interaction.response.send_message(
+            skullboard_info, embed=embed, allowed_mentions=AllowedMentions().none()
+        )
+
+    @app_commands.command(name="rank", description="Provides ranking of users")
+    async def rank(self, interaction: Interaction):
+        try:
+            rankings = await self.db.get_user_rankings()
+            if not rankings:
+                await interaction.response.send_message(
+                    "Database error fetching user rankings - check the logs."
+                )
+                return
+
+            # Warning: description in embed cannot be longer than 2048 characters
+            msg = ["### Top Users of All-Time:\n"]
+            for user_id, frequency in rankings[:10]:
+                # Format the rankings into a readable message
+                line = f"💀 {frequency} : <@!{user_id}>"
+                msg.append(line)
+            msg = "\n".join(msg)
+
+            embed = Embed(
+                title="💀💀💀   TOP SKULLERS   💀💀💀",
+                colour=LIGHT_GREY,
+                description=msg,
+            )
+
+            await interaction.response.send_message(
+                embed=embed, allowed_mentions=AllowedMentions().none()
+            )
+
+        except Exception as e:
+            await interaction.response.send_message(f"An error occurred: {str(e)}")
+
+    @app_commands.command(
+        name="hof", description="Hall of fame rankings for the top posts"
+    )
+    async def hof(self, interaction: Interaction):
+        try:
+            hof_entries = await self.db.get_HOF()
+            if not hof_entries:
+                await interaction.response.send_message(
+                    "Database error fetching Hall of Fame - check the logs."
+                )
+                return
+
+            # Warning: description in embed cannot be longer than 2048 characters
+            msg = ["### Top Posts of All-Time:"]
+            for post_id, user_id, channel_id, day, frequency in hof_entries[:10]:
```

Comment:
Remove `day` since it's not used.

Reply:
I have to leave it in there for iteration purposes. There will be a mismatched shape error otherwise.

Reply:
I guess you could use `_` to just mark the variable as unused

Reply:
I would rather not do that. In the future `day` may be used. Worth including for self-documentation.

Reply:
In that case it fine, but leave a comment in the code saying that.